yktracec/src/stackmap_oll_plugin.cc

```diff
@@ -0,0 +1,59 @@
+// LLVM ORC Object Linking Layer plugin for preserving and locating stackmaps.
+
+#include "stackmap_oll_plugin.h"
+
+#ifdef __linux__
```

Comment:
Presumably we need an `#else #error unsupported platform` or similar, to stop "undefined variable SMSectionName" confusion?

Reply:
Fixed in 21d0acad